### PR TITLE
Removed whitespaces

### PR DESCRIPTION
**Pull request by toin0u**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,5 +31,5 @@ Alertify is licensed under MIT http://www.opensource.org/licenses/MIT
 
 ### Copyright
 
-Copyright (c) 2012, Fabien Doiron
```
Comment (fabien-d) [inline README.md]: The 2 trailing spaces acts as a line break for markdown.

**Comment by fabien-d** [comment]
rest looks good. I need to pay more attention to this!

File: README.md (hunk 1 above)
Comment (toin0u) [inline README.md]: Sorry for that ! :-/

**Comment by toin0u** [comment]
Done :)

**Pull request merged**